src/main/java/quinn/task/Deadline.java

```diff
@@ -0,0 +1,23 @@
+package quinn.task;
+
+public class Deadline extends Task {
+    private final String byDateTimeInput;
```

Comment:
Perhaps using dueDateTime is more descriptive and concise here. Furthermore adding the word 'input' in the naming dosent't add significant value and can make the code more verbose.